Gruntfile.js

```diff
@@ -447,10 +466,21 @@ module.exports = function( grunt ) {
 					packageManager: "npm"
 				}
 			}
+		},
+		"start-selenium-server": {
+			dev: {
+				options: {
+					downloadUrl: "https://selenium-release.storage.googleapis.com/2.45/" +
+						"selenium-server-standalone-2.45.0.jar",
+					downloadLocation: "node_modules/grunt-selenium-server/",
+					serverOptions: serverOptions,
+					systemProperties: {}
+				}
+			}
```

Comment:
might fault since it was not in the gist i referred you to but this should include
```js
"stop-selenium-server": {
    dev: {}
},
```